Front-end/parquick/src/components/MainDriver/MainDriver.jsx

```diff
@@ -41,7 +49,7 @@ function MainDriver(): React.MixedElement {
             </div>
         </div>
         <div className="map">
-            <Map/>
+            { isLoaded? <Map isLoaded={isLoaded}/> : <></>}
```

Comment:
You could also do
```
{isLoaded && <Map isLoaded />}
```
You can omit assigning a value to a boolean prop if that value is either `true` or `false`.